core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/resources.js

```diff
@@ -144,7 +144,7 @@ function onCheckChanged(checkBox){
         var allCheckBoxes = document.querySelectorAll('.token-to-revoke');
         for(var i = 0; i < allCheckBoxes.length; i++){
             var checkBox = allCheckBoxes[i];
```

Comment:
Why do not use `const` since the variable is never changed?

Reply:
Consistency with the rest of the core javascript :) Rewriting `var` into `const`/`let` should be part of a global refactoring.

Reply:
yes and no, it will fix the bug using it here I assume, so breaking consistency here is because addressing a bug, correct?

Reply:
As mention in the PR description / solution paragraph:
> Either using `let` instead of `var` or using the `this` that corresponds to the checkbox in this case.

(let or const actually)
I chose the `this` approach especially for the consistency, taking advantage of the value of `this` in that context.

TBH if the `this` approach was not available, I would have rewritten the code to have a function scope, just to avoid introducing `const`.